nipy/algorithms/registration/groupwise_registration.py

```diff
@@ -737,9 +751,17 @@ def _init(self, images, tr, slice_times, slice_info, affine_class):
         else:
             time_interp = True
         self.slice_times = slice_times
-        self.tr = tr
         if tr is None:
-            raise ValueError('Repetition time cannot be None')
+            tr = tr_from_header(images)
+            if tr == 1:
+                raise ValueError('A TR of 1 was found in the header. '\
```

Comment:
You don't need the line continuation here, Python will combine the strings without them.